pages/homePage/homePage.html

```diff
@@ -0,0 +1,160 @@
+                        <div class="dui-util-clr-prim-p2">
+                            <svg xmlns="http://www.w3.org/2000/svg"
+                                class="btn__icon icon icon-tabler icon-tabler-shopping-cart" width="56" height="56"
+                                viewBox="0 0 24 24" stroke-width="1.5" stroke="#9A9A9A" fill="none"
+                                stroke-linecap="round" stroke-linejoin="round">
+                                <path stroke="none" d="M0 0h24v24H0z" fill="none" />
+                                <circle cx="6" cy="19" r="2" />
+                                <circle cx="17" cy="19" r="2" />
+                                <path d="M17 17h-11v-14h-2" />
+                                <path d="M6 5l14 1l-1 7h-13" />
+                            </svg>
+                            <p class="dui-util-fw-bld">Card</p>
+                        </div>
+                    </button>
+                </div>
+            </nav>
+        </section>
+
+        <section class="page-sidebar">
+
+        </section>
+
+        <section class="page-main">
+            <main class="main-content">
+                <header class="main-content__header">
+                    <!-- Slideshow container -->
+                    <div class="slideshow-container">
+
+                        <!-- Full-width images with number and caption text -->
+                        <div class="mySlides fade">
+                            <div class="numbertext">1 / 3</div>
+                            <img src="/assets/images/carousal_1.jpg" style="width:100%">
+                            <div class="text">Caption Text</div>
+                        </div>
+
+                        <div class="mySlides fade">
+                            <div class="numbertext">2 / 3</div>
+                            <img src="/assets/images/carousal_2.jpg" style="width:100%">
+                            <div class="text">Caption Two</div>
+                        </div>
+
+                        <div class="mySlides fade">
+                            <div class="numbertext">3 / 3</div>
+                            <img src="/assets/images/carousal_1.jpg" style="width:100%">
+                            <div class="text">Caption Three</div>
+                        </div>
+
+                        <!-- Next and previous buttons -->
+                        <a class="prev" onclick="plusSlides(-1)">&#10094;</a>
+                        <a class="next" onclick="plusSlides(1)">&#10095;</a>
+
+                        <br>
+                        <!-- The dots/circles -->
+                        <div class="carousal_navigator" style="text-align:center">
+                            <span class="dot" onclick="currentSlide(1)"></span>
+                            <span class="dot" onclick="currentSlide(2)"></span>
+                            <span class="dot" onclick="currentSlide(3)"></span>
```

Comment:
Don't use on click do this in js using id's

Reply:
+1

Reply:
Done.